README.md

```diff
@@ -103,6 +103,12 @@ class UrlGenerator < Alephant::Broker::LoadStrategy::HTTP::URL
 end
 ```
 
+##### Revalidate Strategy
+
+```ruby
```

Comment:
On an additional note, it might be an idea to create a sequence diagram when this PR is near merge and then stick it into the documentation repo + a link to it within the PR description as it'll make comprehending a lot of the moving parts a lot easier